src/z3c/autoinclude/utils.txt

```diff
@@ -27,7 +27,7 @@ by having been extended by nested packages) it should find a package::
 
     >>> import enolp
     >>> distributionForPackage(enolp)
-    enolp.ppa.foo 0.1 (...enolp.ppa.foo-0.1...egg)
+    enolp.ppa.bar 0.1 (...enolp.ppa.bar-0.1...egg)
```

Comment:
Is this deterministic now?  If not maybe we can pick the 1st one alphabetically?